locale/en_US/manager.xml

```diff
@@ -547,6 +547,7 @@
 	<message key="manager.navigationMenus.unassignedMenuItems">Unassigned Menu Items</message>
 	<message key="manager.navigationMenus.form.navigationMenuArea">Active Theme Navigation Areas</message>
 	<message key="manager.navigationMenus.form.navigationMenuAreaMessage">Select a navigation area</message>
+	<message key="manager.navigationMenus.form.lastOfArea">You can't bind a navigation menu to an area that is bound to another navigation menu.</message>
```

Comment:
Update the locale key to better match what the error is, and change the text to: "A navigation menu is already assigned to this area."

Reply:
OK